core/vm/access_list_tracer.go

```diff
@@ -96,7 +96,7 @@ func (al accessList) equal(other accessList) bool {
 func (al accessList) accessList() types.AccessList {
 	acl := make(types.AccessList, 0, len(al))
 	for addr, slots := range al {
-		tuple := types.AccessTuple{Address: addr}
+		tuple := types.AccessTuple{Address: addr, StorageKeys: []common.Hash{}}
```

Comment:
This fixes an issue where sometimes the AL tracer would produce an accesslist with nil StorageKeys. This would then fail the marshaling/unmarshaling